### PR TITLE
Edit the RuntimeException message

### PR DESCRIPTION
**Pull request by parnissolle**
Edit the message sent when extracting the archive to better match the user's request.

### CODE REVIEW

**Comment by taylorotwell** [comment]
Seems like a lot of other changes here.

File: src/NewCommand.php [hunk 1]
```diff
@@ -236,6 +228,29 @@ protected function getVersion(InputInterface $input)
         return 'master';
     }
 
+    /**
+     * Get the filename that should be downloaded.
+     *
+     * @param  string  $version
+     * @return string
+     */
+    protected function getFilename($version)
```
Comment (parnissolle) [inline src/NewCommand.php:237]: A method to get the archive filename

File: src/NewCommand.php [hunk 2]
```diff
@@ -133,23 +136,11 @@ protected function makeFilename()
      * Download the temporary Zip to the given file.
      *
      * @param  string  $zipFile
-     * @param  string  $version
+     * @param  string  $filename
      * @return $this
      */
```
Comment (parnissolle) [inline src/NewCommand.php:141]: The version is now passed to the `getFilename` method to properly get the required filename

File: src/NewCommand.php [hunk 3]
```diff
@@ -59,8 +59,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $output->writeln('<info>Crafting application...</info>');
```
Comment (parnissolle) [inline src/NewCommand.php:61]: We now get the file name from version here, and pass it to `download` and `extract` methods.

File: src/NewCommand.php [hunk 4]
```diff
@@ -162,16 +153,17 @@ protected function download($zipFile, $version = 'master')
...
-    protected function extract($zipFile, $directory)
+    protected function extract($zipFile, $directory, $filename)
     {
         $archive = new ZipArchive;
 
         $response = $archive->open($zipFile, ZipArchive::CHECKCONS);
 
         if ($response === ZipArchive::ER_NOZIP) {
-            throw new RuntimeException('The zip file could not download. Verify that you are able to access: http://cabinet.laravel.com/latest.zip');
+            throw new RuntimeException("The zip file could not download. Verify that you are able to access: http://cabinet.laravel.com/{$filename}");
```
Comment (parnissolle) [inline src/NewCommand.php:166]: Then here we are... the purpose to the pull request was to unify the way we render the file name to the user.

**Pull request closed without merging**